package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Jan 21 07:24:03 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- bsc#1156986
+  - Fix Yast::Logger dependency in Y2Network::VirtualizationConfig
```

Comment:
well, better to have here real user use case. This is more developer oriented message. I expect something like "fix crash when ..."

Reply:
Ok, updated according to your suggestions.